
Using device: cuda
Traceback (most recent call last):
  File "train.py", line 189, in <module>
    train(net, args)
  File "train.py", line 56, in train
    for i, batch in enumerate(dataloader):
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torch\utils\data\dataloader.py", line 619, in __next__
    data = self._next_data()
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torch\utils\data\dataloader.py", line 1351, in _next_data
    return self._process_data(data)
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torch\utils\data\dataloader.py", line 1377, in _process_data
    data.reraise()
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torch\_utils.py", line 457, in reraise
    raise exception
cv2.error: Caught error in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torch\utils\data\_utils\worker.py", line 287, in _worker_loop
    data = fetcher.fetch(index)
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torch\utils\data\_utils\fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torch\utils\data\_utils\fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "D:\hammer\folder\flood_unet\datasets\dataloader.py", line 29, in __getitem__
    image_vv, image_vh, label = self.transform((image_vv, image_vh, label))
  File "C:\Users\Administrator\.conda\envs\torch\lib\site-packages\torchvision\transforms\transforms.py", line 95, in __call__
    img = t(img)
  File "D:\hammer\folder\flood_unet\datasets\dataloader.py", line 127, in __call__
    img1 = cv2.resize(img1, self.size)
cv2.error: OpenCV(4.5.4) :-1: error: (-5:Bad argument) in function 'resize'
> Overload resolution failed:
>  - Can't parse 'dsize'. Sequence item with index 0 has a wrong type
>  - Can't parse 'dsize'. Sequence item with index 0 has a wrong type